src/entities/activity/ui/ItemCardButtons.tsx

```diff
@@ -1,80 +1,90 @@
+  const isAudioPlayerItem = currentItem?.responseType === "audioPlayer"
 
-  if (isOnePageAssessment) {
-    return (
-      <div className={classNames("no-gutters", "d-flex", "flex-row", "justify-content-around")}>
-        {(isSubmitShown && (
-          <BasicButton
-            variant="outline-dark"
-            size="lg"
-            onClick={onSubmitButtonClick}
-            className={classNames("mb-2", "navigator-button")}>
-            {submitLabel}
-          </BasicButton>
-        )) || <div />}
+  const isItemWithoutAnswer = isMessageItem || isAudioPlayerItem
 
-        {config.isSkippable && (
-          <BasicButton
-            disabled={!config.isNextDisabled}
-            variant="outline-dark"
-            size="lg"
-            className={classNames("mb-2", "navigator-button")}>
-            {t("Consent.skip")}
-          </BasicButton>
-        )}
-      </div>
-    )
+  const config: ItemCardButtonsConfig = {
+    isNextDisabled: isItemWithoutAnswer ? false : !currentItem?.answer || !currentItem.answer.length,
+    isSkippable: currentItem?.config.skippableItem || isAllItemsSkippable,
+    isBackShown: hasPrevStep && !currentItem?.config.removeBackButton,
   }
 
+  const nextLabel = config.isNextDisabled && config.isSkippable ? t("Consent.skip") : t("Consent.next")
+  const submitLabel = t("submit")
+
+  const nextOrSubmitButtonLabel = isSubmitShown ? submitLabel : nextLabel
+
   return (
-    <div className={classNames("no-gutters", "d-flex", "flex-row", "justify-content-around")}>
+    <Box
+      display="flex"
+      flex={1}
+      justifyContent="space-between"
+      alignItems="center"
+      margin="0 auto"
+      padding={greaterThanSM ? "0px 24px" : "0px 16px"}
+      maxWidth="900px">
       {(config.isBackShown && (
-        <BasicButton
-          variant="outline-dark"
-          size="lg"
+        <Button
+          variant="outlined"
           onClick={onBackButtonClick}
-          className={classNames("mb-2", "navigator-button")}>
+          sx={{ borderRadius: "100px", padding: "10px 24px", width: greaterThanSM ? "200px" : "120px" }}>
           {t("Consent.back")}
-        </BasicButton>
-      )) || <div />}
+        </Button>
+      )) || <div></div>}
 
-      <BasicButton
-        variant="outline-dark"
-        size="lg"
-        disabled={!config.isSkippable && config.isNextDisabled}
-        className={classNames("mb-2", "navigator-button")}
-        onClick={isSubmitShown ? onSubmitButtonClick : onNextButtonClick}>
-        {isSubmitShown ? submitLabel : nextLabel}
-      </BasicButton>
-    </div>
+      {isLoading ? (
+        <Button
+          variant="contained"
+          sx={{ borderRadius: "100px", padding: "10px 24px", width: greaterThanSM ? "200px" : "120px" }}
+          disabled={isLoading}
+          onClick={isSubmitShown ? onSubmitButtonClick : onNextButtonClick}>
+          <CircularProgress size={25} sx={{ color: Theme.colors.light.onPrimary }} />
```

Comment:
Maybe it'd be a better idea to just leave one <Button> component and move the conditional rendering to the child of the button:
```
<Button
          variant="contained"
          sx={{ borderRadius: "100px", padding: "10px 24px", width: greaterThanSM ? "200px" : "120px" }}
          disabled={!config.isSkippable && config.isNextDisabled}
          onClick={isSubmitShown ? onSubmitButtonClick : onNextButtonClick}>
          {
             isLoading 
               ? nextOrSubmitButtonLabel
               : <CircularProgress size={25} sx={{ color: Theme.colors.light.onPrimary }} />
           }
</Button>

Reply:
Done